src/exchange/tradingPair/TradingPairView.tsx

```diff
@@ -28,20 +30,47 @@ export class TradingPairView extends React.Component<TradingPairsProps, { showMe
           exact={true}
           to={{ pathname, state: { pair } }}
           activeClassName={styles.active}
-          className={styles.dropdownItemLink}
+          className={classnames(styles.dropdownItemLink, styles.pairView)}
         >
-          <TradingPairView.PairView base={pair.base} quote={pair.quote} />
+          <TradingPairView.PairView { ...{ pair, marketsDetailsLoadable } } />
         </NavLink>
       </li>
     );
   }
 
-  public static PairView({ base, quote }: any) {
+  public static PairView({ pair, marketsDetailsLoadable }: { pair: TradingPair, marketsDetailsLoadable: Loadable<MarketsDetails> }) {
+    const { base, quote } = pair;
     return (
-      <div className={styles.pairView}>
-        <div className={styles.pairViewIcon}>{tokens[base].iconCircle}</div>
+      <>
+        <div className={styles.pairViewIcon}>{tokens[base].icon}</div>
         <div className={styles.pairViewCurrency}>{base}</div>
-        <div className={styles.pairViewCurrency}>{quote}</div>
+        <div className={styles.pairViewCurrency}><FormatQuoteToken token={quote} /></div>
+        <WithLoadingIndicatorInline loadable={marketsDetailsLoadable}>
+          {(marketsDetails) => {
+            const { price, priceDiff } = marketsDetails[tradingPairResolver(pair)];
+            return (<>
+              <div className={styles.pairViewIconQuote}>{tokens[quote].icon}</div>
+              <div className={styles.pairViewPrice}>{price &&
+                <FormatPrice value={price} token={quote} />
+              }</div>
+              <div className={styles.pairViewPriceDiff}>{priceDiff &&
+                <BoundarySpan value={priceDiff}>
+                  <FormatPercent value={priceDiff} />
+                </BoundarySpan>
+              }</div>
+            </>);
+          }}
+        </WithLoadingIndicatorInline>
+      </>
+    );
+  }
+
+  public static ActivePairView({ base, quote }: any) {
+    return (
+      <div className={styles.activePairView}>
+        <div className={styles.activePairViewIcon}>{tokens[base].iconCircle}</div>
+        <div className={styles.activePairViewCurrency}>{base}</div>
+        <div className={styles.activePairViewCurrency}><FormatQuoteToken token={quote} /></div>
```

Comment:
Have the mapping of colors somewhere in this file.

Reply:
If it's in this file then it's not reusable